plugins/ProfileExtender/class.profileextender.plugin.php

```diff
@@ -574,17 +574,22 @@ public function utilityController_exportProfiles_create($sender) {
                 ->leftJoin('Rank ra', 'ra.RankID = u.RankID');
         }
 
+        $lowerCaseColumnNames =  array_map('strtolower', $columnNames);
         $i = 0;
         foreach ($fields as $slug => $fieldData) {
+            // Don't overwrite data if there's already a column with the same name.
+            if (in_array(strtolower($slug), $lowerCaseColumnNames)) {
+                continue;
+            }
             // Add this field to the output
-            $columnNames[] = val('Label', $fieldData, $slug);
-
-            // Add this field to the query.
-            $quoted = Gdn::sql()->quote("Profile.$slug");
-            Gdn::sql()
-                ->join('UserMeta a'.$i, "u.UserID = a$i.UserID and a$i.Name = $quoted", 'left')
-                ->select('a'.$i.'.Value', '', $slug);
-            $i++;
+                $columnNames[] = val('Label', $fieldData, $slug);
```

Comment:
Your indentation here seems all messed up. Any reason for this?

Reply:
> Your indentation here seems all messed up. Any reason for this?

I'm not sure if this is a sincere question or not. If it is, the answer is that it was simply an oversight. I believe it's fixed now.